lib/OrdinaryDiffEqFIRK/src/firk_perform_step.jl

```diff
@@ -1369,21 +1369,16 @@ end
 
     J = calc_J(integrator, cache)
 
-    #if u isa Number
-    #    LU1 = Complex(-γdt * mass_matrix + J)
-    #    LU2 = -(αdt[1] + βdt[1] * im) * mass_matrix + J
-    #else
-        LU1 = lu(-γdt * mass_matrix + J)
-        LU2 = lu(-(αdt[1] + βdt[1] * im) * mass_matrix + J)
-    #end
-    LU = [LU2 for _ in 1:(num_stages + 1) ÷ 2]
+    LU2 = Vector{Complex{typeof(u)}}(undef,  (num_stages - 1) ÷ 2)
```

Comment:
This is wrong. In the `Vector` case, the `lu` call means that this has a different type.

Reply:
should be fixed